Source/ASDisplayNode+Yoga.mm

```diff
@@ -58,7 +58,7 @@ - (void)setYogaChildren:(NSArray *)yogaChildren
 
 - (NSArray *)yogaChildren
 {
-  return _yogaChildren;
+  return [_yogaChildren copy] ?: @[];
```

Comment:
We already lock on the setter, why not go ahead and lock here and declare the property `atomic`?

Reply:
Because this diff was copied from an 11th-hour internal hotfix! But you're right, we should do it now. @maicki mind making the change and landing it with this?

Reply:
I knew I'd been here before...